src/Components/FollowSidebar/FollowSidebar.jsx

```diff
@@ -0,0 +1,38 @@
+import { useSelector } from "react-redux";
+import "./FollowSidebar.css";
+export const FollowSidebar = () => {
+  const { users } = useSelector((store) => store.users);
+  console.log(users);
+  return (
+    <div className="follow-sidebar">
+      <h1>Who to Follow</h1>{" "}
+      {users.length > 0 ? (

```

Comment:
If user is `undefined`, the code would break, here you can use `users?.length`

Reply:
sure I have updated them